index.js

```diff
@@ -389,6 +390,37 @@ function commonJSLookup(options) {
   return result;
 }
 
+function vueLookup(options) {
+  const dependency = options.dependency;
+
+  if (!dependency) {
```

Comment:
This whole block is probably better written with a `switch` statement.

Reply:
it cannot be original table lookup switch because those are endsWith statements, they require runtime evaluation